nautobot_ssot/models.py

```diff
@@ -161,9 +161,9 @@ class SyncLogEntry(BaseModel):
     synced_object_id = models.UUIDField(blank=True, null=True)
     synced_object = GenericForeignKey(ct_field="synced_object_type", fk_field="synced_object_id")
 
-    object_repr = models.CharField(max_length=200, blank=True, default="", editable=False)
+    object_repr = models.TextField(blank=True, default="", editable=False)
```

Comment:
does it work well for Mysql? (I don't think so)

Reply:
What's the potential concern with MySQL? Performance?

Reply:
`TextField` is not supported. I remember fixing it here: https://github.com/nautobot/nautobot-plugin-circuit-maintenance/pull/159/files

Reply:
Good recollection. IIRC `TextField` is supported in MySQL, but you can't do a `TextField(unique=True)` or use it as part of a `unique_together` constraint.

Reply:
https://docs.djangoproject.com/en/3.2/ref/databases/#textfield-limitations